examples/nested-folders/src/Folder.js

```diff
@@ -78,29 +78,23 @@ function view(state$, childrenVDOM$) {
     })
 }
 
-function Children(sources) {
-  const array$ = sources.onion.state$
-
-  const childrenSinks$ = array$.map(array =>
-    array.map((item, index) => isolate(Folder, index)(sources))
-  )
-
-  const childrenReducer$ = childrenSinks$
-    .compose(pick('onion'))
-    .compose(mix(xs.merge))
-
-  const childrenVDOM$ = childrenSinks$
-    .compose(pick('DOM'))
-    .compose(mix(xs.combine))
+function Children(sources = {}) {
```

Comment:
The function wrapper is not needed here: You have:
```
function X(something) {
    const somethingElse = ...;
    return somethingElse(something)
}
```
I would remove that wrapper:
```js
const Children = makeCollection({
  /* ... */
});
```